### PR TITLE
OP-418 remove unnecessary if test

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Found with SpotBugs.  

Remove unnecessary `if` statement as code is duplicated in both branches.

### CODE REVIEW

File: src/main/java/org/isf/admission/gui/PatientDataBrowser.java [hunk 1]
```diff
@@ -210,14 +210,8 @@ private JPanel getPatientDataPanel(){
...
-		if (isMalnutrition){
-			patientData.add(ps.getPatientCompleteSummary(), BorderLayout.WEST);
-		}
-		else {
-			//patientData.add(ps.getPatientDataPanel(), BorderLayout.WEST);			
-			patientData.add(ps.getPatientCompleteSummary(), BorderLayout.WEST);
-		}
-		
+		patientData.add(ps.getPatientCompleteSummary(), BorderLayout.WEST);
+
```
Comment (mwithi) [inline src/main/java/org/isf/admission/gui/PatientDataBrowser.java:214]: Safe to remove

**Review by mwithi — CHANGES_REQUESTED** [review]
See comments

File: src/main/java/org/isf/lab/gui/LabEditExtended.java [hunk 2]
```diff
@@ -417,11 +417,7 @@ private JComboBox getPatientComboBox(String s) {
 		//String key = s;
 		PatientBrowserManager patBrowser = Context.getApplicationContext().getBean(PatientBrowserManager.class);
 		try {
-			if (insert){
-				pat = patBrowser.getPatient();
-			} else  {
-				pat = patBrowser.getPatient();
-			}
+			pat = patBrowser.getPatient();
```
[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply src/main/java/org/isf/lab/gui/LabEditExtended.java:420]: I added comments to the JIRA issue.

[the remark this answers is not in the document]
Comment (dbmalkovsky) [reply src/main/java/org/isf/lab/gui/LabEditExtended.java]: Changes made

[the remark this answers is not in the document]
Comment (mwithi) [reply src/main/java/org/isf/lab/gui/LabEditExtended.java]: Thanks a lot, also for the comment! 🙌

**Review by mwithi — APPROVED** [review]

**Pull request merged**